social-network/src/Articles.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './ArticleData.css';
+import './Articles.css';
+import YourArticleData from './YourArticleData';
+import MissedArticleData from './MissedArticleData';
+
+class Articles extends Component {
+
+  render() {
+
+    return (
+      <div className='container'>
+        <h1 className='articleSectionHeader'>For you</h1>
+        <div className='articlesContainer'>
+          <div className='yourArticles'>
+            <YourArticleData index={0} className='yourArticle1' />
```

Comment:
While this works, it doesn't extend very well if the article list grows/shrinks. Instead, you'll want to use `.map` on the articles data and map each article to the component.